### DISCUSSION TITLE
not enough dimentions

### DISCUSSION THREAD

**Discussion by mirza11227** [Jannaiz/Cosmic-Chess#21] [Ideas]
mooooaaaaaaaarrrrrrr

**Comment by Jannaiz** [comment]
So what is your idea, that sounds like a problem?
But, we are working on it.
Should we already create an issue for that @AtlantiaKing ?

**Comment by AtlantiaKing** [comment]
@mirza11227 What a great idea! We really do need more dimensions.
@Jannaiz Sure thing, I'll look into it.
